test/inductor/test_aot_inductor.py

```diff
@@ -4156,10 +4156,9 @@ class AOTInductorTestABICompatibleCpu(TestCase):
 
 
 @unittest.skipIf(sys.platform == "darwin", "No CUDA on MacOS")
-@unittest.skipIf(not HAS_CUDA, "No CUDA")
```

Comment:
I think it should be `skipIf(not HAS_GPU)` rather than removing it directly because the test class is only dedicated to GPU device.

Reply:
The latest is https://github.com/pytorch/pytorch/pull/141961/files#diff-6b55ea9919f49e90d28a14bbacc3793c4b3ba932603c141b9452d333c4585642R4228.